openassessment/xblock/static/js/src/studio/oa_edit_listeners.js

```diff
@@ -142,7 +142,7 @@ OpenAssessment.StudentTrainingListener.prototype = {
         if (invalidated) {
             this.displayAlertMsg(
                 gettext("Option Deleted"),
-                gettext("You've deleted an option. The system has removed that option from its criterion in the sample responses in the Student Training step. You may have to select a new option for the criterion.")
+                gettext("You've deleted an option. The system has removed that option from its criterion in the sample responses in the Learner Training step. You may have to select a new option for the criterion.")
```

Comment:
"You've deleted..." -> "You have deleted..."
"The system has removed that option..." -> "That option has been removed from its criterion..."
"You may have to select..." -> "You might have to select..."